### PR TITLE
DUPLO-21167: add new airflow versions and remove deprecated versions

### PR DESCRIPTION
**Pull request by pravin-duplocloud**
## Overview

DUPLO-21167: add new airflow versions and remove deprecated versions

## Summary of changes

This PR does the following:

- add new airflow versions and remove deprecated versions
- add new instance size

## Testing performed

- [ ] Using unit tests
- [X] Manually, on my local system
- [X] Manually, on a remote test system

## Describe any breaking changes

- None

___

### **PR Type**
enhancement, documentation

___

### **Description**
- Added new Airflow versions to the MWAA environment schema and removed deprecated ones.
- Introduced new instance sizes (`mw1.xlarge`, `mw1.2xlarge`) for MWAA environments.
- Updated documentation to reflect changes in instance sizes.
- Updated CHANGELOG to include enhancements and documentation improvements.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>resource_duplo_aws_mwaa_environment.go</strong><dd><code>Update Airflow versions and instance sizes in MWAA schema</code></dd></summary>
<hr>

duplocloud/resource_duplo_aws_mwaa_environment.go

<li>Added new Airflow versions to the validation list.<br> <li> Introduced new instance sizes for environment classes.<br> <li> Updated schema to force new resource on version change.<br>

</details>

  </td>
  <td><a href="https://github.com/duplocloud/terraform-provider-duplocloud/pull/694/files#diff-6a469d4148610b03e001d2e23b76365f3233f3ddb547161214c8de3bb599207a">+6/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>CHANGELOG.md</strong><dd><code>Update CHANGELOG with Airflow and instance size changes</code>&nbsp; &nbsp; </dd></summary>
<hr>

CHANGELOG.md

<li>Documented addition of new Airflow versions.<br> <li> Documented introduction of new instance sizes.<br> <li> Improved organization of sections.<br>

</details>

  </td>
  <td><a href="https://github.com/duplocloud/terraform-provider-duplocloud/pull/694/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed">+7/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>aws_mwaa_environment.md</strong><dd><code>Document new instance sizes for MWAA environments</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

docs/resources/aws_mwaa_environment.md

- Updated documentation for new instance sizes.

</details>

  </td>
  <td><a href="https://github.com/duplocloud/terraform-provider-duplocloud/pull/694/files#diff-2c4d96ee3aee8451bac1cc346c88b3b2006ad1f28639ac0282e78c979c33f0cf">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

**Comment by zafarabbas** [comment]
Task linked: [DUPLO-21167 Update Airflow version to all supported](https://app.clickup.com/t/8655600/DUPLO-21167)

File: duplocloud/resource_duplo_aws_mwaa_environment.go [hunk 1]
```diff
@@ -83,14 +83,18 @@ func duploMwaaAirflowSchema() map[string]*schema.Schema {
 			Type:        schema.TypeString,
 			Computed:    true,
 			Optional:    true,
+			ForceNew:    true,
+			ValidateFunc: validation.StringInSlice([]string{
+				"2.4.3", "2.5.1", "2.6.3", "2.7.2", "2.8.1", "2.9.2",
+			}, false),
 		},
 		"environment_class": {
 			Description: "Environment class for the cluster. Possible options are `mw1.small`, `mw1.medium`, `mw1.large`.",
```
Comment (sp1676) [inline duplocloud/resource_duplo_aws_mwaa_environment.go:92]: Description should be changed as well.

**Review by sp1676 — CHANGES_REQUESTED** [review]

File: duplocloud/resource_duplo_aws_mwaa_environment.go (hunk 1 above)
Comment (tahir-duplo) [reply duplocloud/resource_duplo_aws_mwaa_environment.go]: Done

**Review by tahir-duplo — APPROVED** [review]
LGTM

**Pull request merged**